torch/autograd/gradcheck.py

```diff
@@ -20,6 +20,17 @@ class GradcheckError(RuntimeError):
     pass
 
 
+
+def _is_sparse_compressed_tensor(obj):
+    return is_tensor_like(obj) and obj.layout in {torch.sparse_csr, torch.sparse_csc,
+                                                  torch.sparse_bsr, torch.sparse_bsc}
+
+
+def _is_sparse_any_tensor(obj):
+    return is_tensor_like(obj) and obj.layout in {torch.sparse_coo, torch.sparse_csr, torch.sparse_csc,
```

Comment:
nit: most of this list can be deduplicated with the one above.

Reply:
Fixed. Also, the call to `is_tensor_like` turns out to be unnecessary because these functions are used in the context where the inputs are always tensors.